openstack-cloud/src/main/java/jenkins/plugins/openstack/compute/JCloudsSlaveTemplate.java

```diff
@@ -428,6 +435,46 @@ public ListBoxModel doFillNetworkIdItems(@RelativePath("..") @QueryParameter Str
             return m;
         }
 
+        public ListBoxModel doFillFloatingIpPoolIdItems(@RelativePath("..") @QueryParameter String identity,
```

Comment:
It is a best practice for methods that are exposed for jelly/databinding and needs to be `public`. This clearly declares that it is not a part of plugin API.